library/src/test/java/io/constructor/core/ConstructorIoTrackingTest.kt

```diff
@@ -1105,4 +1204,46 @@ class ConstructorIoTrackingTest {
         val request = mockServer.takeRequest(10, TimeUnit.SECONDS)
         assertEquals(null, request)
     }
+
+    @Test
+    fun behaviorEndpointWithPiiShouldBeOmitted() {
+        val mockResponse = MockResponse().setResponseCode(204)
+
+        for (email in emailPii) {
+            mockServer.enqueue(mockResponse)
+            val observer = ConstructorIo.trackSearchResultsLoadedInternal(email, 10).test()
```

Comment:
Let's change the tests to use `ConstructorIo.trackSearchSubmitInternal` so we can test to make sure that PII is being obfuscated in both the path and query params. We may want to have a stricter assertion here instead of just making sure that it contains `email_omitted` since it can show up in multiple places.